packages/fluentui/docs/src/index.ejs

```diff
@@ -1,129 +1,139 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <base href='<%= __BASENAME__ %>'>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <link rel="shortcut icon" type="image/x-icon" href='<%= __PROD__ ? 'https://fabricweb.azureedge.net/fabric-website/assets/images/fluent-ui-logo.png' : 'https://fabricweb.azureedge.net/fabric-website/assets/images/fluent-ui-logo-dev.png' %>'/>
-  <title>Fluent UI</title>
-  <script>
-    // Redirect to HTTPs
-    // We need poor-mans client side redirects because we persist app router urls in localStorage
-    if (location.hostname === 'fluentsite.z22.web.core.windows.net' && location.protocol !== 'https:') {
-      location.href = location.href.replace('http:', 'https:')
-    }
-    // Apply gh-pages SPA redirect that was applied in 404.html
-    // ----------------------------------------
-    (function() {
-      var redirect = sessionStorage.redirect
-      delete sessionStorage.redirect
-      if (redirect && redirect !== location.href) {
-        history.replaceState(null, null, redirect)
+  <head>
```

Comment:
The indenting makes it kinda hard to diff, are there any other changes in this files besides that ?

Reply:
I know - it happened automatically.  The only change is to the font-family, here:
  body {
        font-family: -apple-system, 'Segoe UI', BlinkMacSystemFont, 'Segoe UI Web', sans-serif;
      }